pages/doc/integrations_tas_alerts.md

```diff
@@ -292,14 +266,14 @@ The default threshold of 35% assumes Diego Cells are spread across three AZs.
 
 Percentage of Diego Cells with at least one chunk of Disk space available to stage an application.
 
-Insufficient free disk on Diego Cells prevents the staging or starting of apps or tasks, resulting in error messages like 
-```
-ERR Failed to stage app: insufficient resources
-```
+Insufficient free disk on Diego Cells prevents the staging or starting of apps or tasks, resulting in error messages such as
+ 
+`ERR Failed to stage app: insufficient resources`
 
-A Diego Cell will not stage an application if it has less than 6 GB remaining, so that is chosen as the default chunk size.
+A Diego Cell will not stage an application if the Cell has less than 6 GB remaining, which is the default chunk size.
```

Comment:
We think we should maybe delete the second half of this sentence (everything after the comma). This specific use of 'chunk size' is only really a concept for this alert and is sort of a holdover from the way we had it before.